app/models/operatingsystems/redhat.rb

```diff
@@ -72,4 +81,33 @@ def pxe_kernel_options(params)
   def minor_version_help
     '0, 6.1810'
   end
+
+  def fedora?
+    name == 'Fedora'
+  end
+
+  def rhel_compatible?
+    name != 'Fedora'
+  end
+
+  # Tells if the OS meets a requirement. The requirement can be specified as rhel version, fedora version or both.
```

Comment:
ACK, I'll remove cross-requirement option.
This would mean that you will have to specify the requirement for fedora and RHEL specifically to work for both:
``` ruby
fedora_39_os.meets_requirement(rhel: 6)
# => false

fedora_39_os.meets_requirement(rhel: 6, fedora: 35)
# => true

rhel_9_os.meets_requirement(fedora: 20)
# => false

rhel_9_os.meets_requirement(fedora: 20, rhel: 7)
# => true
```
How that sounds to you?